spec/services/customer_portal/generate_url_service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CustomerPortal::GenerateUrlService, type: :service do
```

Comment:
RSpec/FilePath: Spec path should end with `customer_portal/generate_url_service*_spec.rb`. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)